ui/app/AppLayouts/Wallet/popups/swap/SwapModalAdaptor.qml

```diff
@@ -271,18 +272,15 @@ QObject {
 
     function sendApproveTx() {
         root.approvalPending = true
-
-        let account = selectedAccountEntry.item
-        let accountAddress = account.address
+        const accountAddress = root.selectedAccount.address
```

Comment:
we should add null/undefined check when using selectedAccount

Reply:
OK, can add

Reply:
Just used the account address directly, we don't have to take the detour :facepalm: